### ISSUE TITLE
SVG styles missing in hyperHTML 2.0.0

### ISSUE THREAD

**Issue by albertosantini** [WebReflection/hyperHTML#142]
Congrats for the new release `2.0.0`. :)

Only updating the lib, I noticed the styles for a SVG node are missing.

I have been doing something like the following pseudo code:
```
...
const svg = d3.select("... my selector ...");
const node = svg.node();
...
node.style.height="10px";
```

From
```
<svg class="sl mw3" style="height: 10px; stroke: red;"><path d="M0,0L16,0L32,0L48,10L64,10"></path></svg>
```
to
```
<svg class="sl mw3"></svg>
```
If no bell rings, I can prepare a repro test.

The weird thing, one complex OHLC chart works fine, and the sparkline one doesn't.

**Comment by WebReflection** [comment]
Please provide the smallest reproducible issue in a code pen or similar, thanks

**Comment by WebReflection** [comment]
Btw there is a change for the style attribute and that might be the cause.

However, having tests is good to avoid regressions next update :-;

**Comment by WebReflection** [comment]
@albertosantini can you please double check the current version in master? I might have introduced a regression with the style, but it should've been fixed now.

I will release the new version once you confirm everything is now fine.

Thank You!

**Comment by WebReflection** [comment]
FYI I've done an "_emergency release_" that addresses possible SVG issues with `style` attribute **and** make the object/string based behavior consistent with regular DOM.

**Comment by albertosantini** [comment]
It seems the issue was due to a bug on my side.
It was an error initializing the selector with a wrong attribute element value.

Really I don't understand how my code worked correctly with the v1 release.
Sorry for the noise, even if it seems you improved that part. :)

Today I took the opportunity to test `2.0.1` in a long run test, using my financial pet app Argo, and I didn't notice any problem in the memory management. My to cents.

Good job and thanks again for the effort.
